src/components/Button.js

```diff
@@ -0,0 +1,8 @@
+import PropTypes from 'prop-types';
+
+function Button({ name }) { // eslint-disable-line no-unused-vars
+}
```

Comment:
- [ ] The name prop should be outputted in a div and also kindly export the component from the module.